gst/edge/edge_common.c

```diff
@@ -0,0 +1,12 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
+/**
+ * Copyright (C) 2022 Samsung Electronics Co., Ltd.
+ *
+ * @file    edge_common.c
+ * @date    01 Aug 2022
+ * @brief   Publish incoming streams
```

Comment:
Update this doxygen entry; I don't think this file is to publish incoming streams.

Reply:
Updated.